modules/admin_manual/pages/configuration/user/user_management.adoc

```diff
@@ -242,3 +242,64 @@ against quotas.
 
 When a user creates a public link share via URL, and allows uploads, any
 uploaded files count against that user’s quota.
+
+== Moving the User Home
+
+Usually, a users home directory is located in the `data/` directory which location is defined in `config.php` and defaults to `/var/www/owncloud/data`. While in smaller installations, the data directory mounts to a physical drive of the server and in bigger installations the data directory is mounted via NFS, this can be suboptimal when having very big installations with thousands of users and/or users having very big space consumtions. This can result in having high load on a single mount point.
```

Comment:
```suggestion
Usually, a users home folder is located in the `data/` directory which location is defined in `config.php` and defaults to `/var/www/owncloud/data`. Usually in smaller installations, the data directory mounts to a physical drive of the server and in bigger installations the data directory is mounted via NFS, this can be suboptimal when having very big installations with thousands of users and/or users having very big space consumption. This can result in having high load on a single mount point.
```